test/integration/patients/patient/patient-flow.js

```diff
@@ -1757,38 +1765,43 @@ context('patient flow page', function() {
   specify('click+shift multiselect', function() {
     cy
       .routeFlow(fx => {
-        fx.data.id = '1';
-
-        const flowActions = _.sample(fx.data.relationships.actions.data, 4);
-
-        _.each(flowActions, (action, index) => {
-          action.id = `${ index + 1 }`;
+        fx.data = mergeJsonApi(testFlow, {
+          relationships: {
+            state: getRelationship(stateInProgress),
+            owner: getRelationship('11111', 'clinicians'),
+          },
         });
 
-        fx.data.relationships.actions.data = flowActions;
-        fx.data.relationships.state.data.id = '33333';
-        fx.data.relationships.owner.data = {
-          id: '11111',
-          type: 'clinicians',
-        };
-
         return fx;
       })
       .routeFlowActions(fx => {
-        fx.data = _.first(fx.data, 3);
-
-        _.each(fx.data, (action, index) => {
-          action.id = `${ index + 1 }`;
-          action.relationships.flow = { data: { id: '1' } };
-          action.relationships.state.data.id = '22222';
-        });
+        fx.data = [
+          getAction({
```

Comment:
maybe `getActions`

Reply:
Updated in https://github.com/RoundingWell/care-ops-frontend/pull/1249/commits/fc0b51aeca4b760a1e42007ce49934791773ca60.